includes/admin/class-wc-admin-dashboard.php

```diff
@@ -105,18 +105,25 @@ private function get_sales_report_data() {
 		 * Show status widget.
 		 */
 		public function status_widget() {
-			include_once dirname( __FILE__ ) . '/reports/class-wc-admin-report.php';
+			include_once dirname( __FILE__ ) . '/reports/class-wc-admin-report-v2.php';
 
-			$reports = new WC_Admin_Report();
+			$is_wc_admin_disabled = apply_filters( 'woocommerce_admin_disabled', false );
+
+			$reports = new WC_Admin_Report_V2();
+
+			$net_sales_link = 'admin.php?page=wc-reports&tab=orders&range=month';
+			$report_data = $is_wc_admin_disabled ? $this->get_sales_report_data() : $reports->get_performance_data();
```

Comment:
`get_performance_data` also returns an `stdClass` with `net_sales` to match how it is used on line 132.
This made for an easier swap.